src/test/scala/org/kibanaLoadTest/scenario/Dashboard.scala

```diff
@@ -124,11 +153,22 @@ object Dashboard {
         .exec(
           http("query gauge data")
             .post("/api/metrics/vis/data")
-            .body(ElFileBody("data/gaugePayload.json"))
+            .body(StringBody(gaugePayload))
             .asJson
             .headers(headers)
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
         )
-    }
+        .foreach(bSearchPayloadSeq, "index"){
+          exec(http("query bsearch ${index}")
```

Comment:
I noticed that number of `bsearch` calls changes time from time. At the moment I'm doing 9 calls with different payload. But what is the right way to replicate it in load testing scenario?
![image](https://user-images.githubusercontent.com/10977896/113872339-15c3e800-97b4-11eb-86f0-6782e2239edb.png)